tests/run-make/crate-name-priority/rmake.rs

```diff
@@ -4,15 +4,17 @@
 // and the compiler flags, and checks that the flag is favoured each time.
 // See https://github.com/rust-lang/rust/pull/15518
 
-use run_make_support::{bin_name, rfs, rustc};
+//@ ignore-cross-compile (relocations in generic ELF against `arm-unknown-linux-gnueabihf`)
+
+use run_make_support::{bin_name, rfs, rustc, target};
 
 fn main() {
-    rustc().input("foo.rs").run();
+    rustc().target(target()).input("foo.rs").run();
```

Comment:
Why adding target(target) when the test ignores cross-compilation? :thinking:

Reply:
Oh it's very simple -- it's because earlier I tried to make see what would fail on cross-compile but then realized it won't really work. I can remove the `target(target())`s for the ignore-cross-compile tests.

Reply:
Removed useless `.target(target())` for non-cross-compile tests.